// Try catch vs throws
public class ExcepEighteen {	

	public static void m1() throws InterruptedException
	{       
				
		for(int i=0;i<=10;i++)
		{
			System.out.println(i);
			Thread.sleep(1000);
		}
		
	public static void main (String[] args)  
	{
		try{
			
	m1 test= new m1();	
		}
		catch(InterruptedException e)
		{
			System.out.println(e);
		}
}
